rbac/internal/specs/openapi.json

```diff
@@ -958,6 +958,847 @@
+                      "type": "array",
+                      "items": {
+                        "type": "object"
+                      },
+                      "example": [
+                        {
+                          "name": "group1",
+                          "admin_default": true,
+                          "system": false,
+                          "platform_default": true,
+                          "tenant": "tenantA"
+                        },
+                        {
+                          "name": "group2",
+                          "admin_default": true,
+                          "system": false,
+                          "platform_default": true,
+                          "tenant": "tenantB"
+                        }
+                      ]
+                    },
+                    "invalid_default_admin_groups_count": {
+                      "type": "integer",
+                      "example": 2
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Invalid method, only 'Delete' and 'Get' are allowed."
+          }
+        }
+      }
+    },
+    "/api/utils/invalid_default_admin_groups ": {
+      "delete": {
+        "tags": [
+          "Utils"
+        ],
+        "summary": "Remove invalid default admin groups",
+        "description": "View method for querying/removing invalid default admin groups.",
+        "operationId": "invalidAdminGroupDel",
+        "responses": {
+          "204": {
+            "description": ""
+          },
+          "405": {
+            "description": "Invalid method, only 'Delete' and 'Get' are allowed."
+          }
+        }
+      }
+    },
+    "/api/utils/ocm_performance": {
+      "post": {
+        "tags": [
+          "Utils"
+        ],
+        "summary": "Run OCM performance tests",
+        "description": "View method for running OCM performance tests.",
+        "operationId": "ocmPerformance",
+        "responses": {
+          "202": {
+            "description": "OCM performance tests are running in a background worker."
+          },
+          "405": {
+            "description": "Invalid method, only 'POST' is allowed."
+          }
+        }
+      }
+    },
+    "/api/utils/reset_imported_tenants": {
+      "get": {
+        "tags": [
+          "Utils"
+        ],
+        "summary": "Get tenants imported via user import job that would be reset.",
+        "description": "Get tenants imported via user import job that would be reset.",
+        "operationId": "getResetImportedTenants",
+        "responses": {
+          "200": {
+            "description": "{count} tenants would be deleted"
+          },
+          "400": {
+            "description": "Invalid limit parameter, must be an integer."
+          }
+        }
+      }
+    },
+    "/api/utils/reset_imported_tenants ": {
+      "delete": {
```

Comment:
if your endpoints supports more methods .. GET + DELETE in this case, you can define it together
pseudocode:
```
    "/api/utils/reset_imported_tenants": {
      "get": { ...
      "delete": { ...
```
you can get an inspiration from other endpoints e.g. [here](https://github.com/RedHatInsights/insights-rbac/blob/f3d33248c962b6896cb84cd641571a97fdc13ffc/rbac/internal/specs/openapi.json#L883)